src/Xamarin.Android.Build.Tasks/Microsoft.Android.Sdk/targets/Microsoft.Android.Sdk.DefaultProperties.props

```diff
@@ -43,7 +42,10 @@
   </PropertyGroup>
   <PropertyGroup Condition=" '$(AndroidApplication)' == 'true' ">
     <SelfContained Condition=" '$(SelfContained)' == '' ">true</SelfContained>
-    <PublishTrimmed Condition=" '$(AndroidLinkMode)' == 'SdkOnly' or '$(AndroidLinkMode)' == 'Full' ">true</PublishTrimmed>
+    <PublishTrimmed Condition=" '$(PublishTrimmed)' == '' and ('$(AndroidLinkMode)' == 'SdkOnly' or '$(AndroidLinkMode)' == 'Full') ">true</PublishTrimmed>
+    <PublishTrimmed Condition=" '$(PublishTrimmed)' == '' and '$(Configuration)' == 'Release' and '$(AndroidLinkMode)' != 'None' ">true</PublishTrimmed>
+    <TrimMode Condition=" '$(TrimMode)' == '' and ('$(AndroidLinkMode)' == 'SdkOnly' or '$(AndroidLinkMode)' == 'Full') ">link</TrimMode>
```

Comment:
Shouldn't the `TrimMode` be set after the next line, where we set `AndroidLinkMode`?

Reply:
Right now this only sets `$(TrimMode)` if the user set `$(AndroidLinkMode)` in their `.csproj` file. It's using the dotnet/sdk default of `CopyUsed`, which maybe is not what we want for mobile?

@rolfbjarne are you using `TrimMode=Link` by default for `Release` builds?

Reply:
@jonathanpeppers the default will most likely be `TrimMode=Link` for device builds (both debug and release), and `TrimMode=Copy` for simulator builds

Reply:
I updated the code and documentation, so we are defaulting to `TrimMode=link` for `Release` builds now.

I think this setting makes more sense for mobile apps.